source/elements/oneTBB/source/algorithms/functions/collaborative_once_flag_cls.rst

```diff
@@ -0,0 +1,32 @@
+.. SPDX-FileCopyrightText: 2019-2021 Intel Corporation
```

Comment:
Same as above - for the new file please use SPDX-FileCopyrightText: 2021 Intel Corporation